### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by NathanTheGr8**
#21

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ app.get('/', function(req, res, next) {
 app.get('/commits', function(req, res, next) {
 
   var scraper = new Xray();
-  scraper('https://github.com/onedr0p/manage-this-node/commits/master',
+  scraper('https://github.com/NathanTheGr8/manage-this-node/commits/master',
```
Comment (onedr0p) [inline app.js]: Please revert this.

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "serve-favicon": "~2.3.0",
     "x-ray": "^2.0.3"
   },
-  "repository": "git://github.com/onedr0p/manage-this-node",
+  "repository": "git://github.com/NathanTheGr8/manage-this-node",
```
Comment (onedr0p) [inline package.json]: And revert this.

File: package.json [hunk 3]
```diff
@@ -34,7 +34,7 @@
   "engines": {
     "node": ">=0.10"
   },
-  "author": "Devin Buhl <devin.kray@gmail.com>",
+  "author": "Nathan Davis <nathanthegreat94@gmail.com>",
```
Comment (onedr0p) [inline package.json]: And revert this.

**Comment by onedr0p** [comment]
You have a lot of cleaning up to do before I can merge this. You're essentially changing the repo in your commits.

File: Readme.md [hunk 4]
```diff
@@ -66,7 +66,7 @@ forever stop ./bin/www
 Goto `localhost:3000` to see the app.
 
 ### Docker
-A dockerfile for this has been provided by [chimpchimp](https://github.com/chimpchimp) [here](https://github.com/chimpchimp/docker-manage-this-node)
+A dockerfile for this has been provided by [nathanthegr8](https://github.com/nathanthegr8) [here](https://github.com/nathanthegr8/docker-manage-this-node)
```
Comment (onedr0p) [inline Readme.md]: And revert this.

**Comment by NathanTheGr8** [comment]
sorry I thought I had gotten most of those already. I forgot to look at diff.

**Pull request closed without merging**